Tests/UnitTests/Misc/SystemInfoTests.swift

```diff
@@ -67,6 +67,26 @@ class SystemInfoTests: TestCase {
         expect(try SystemInfo.withReceiptResult(.nilURL).isSandbox) == false
     }
 
+    func testIsAppleSubscriptionURLWithAnotherURL() {
+        expect(SystemInfo.isAppleSubscription(managementURL: URL(string: "www.google.com")!)) == false
+    }
+
+    func testIsAppleSubscriptionURLWithStandardURL() {
+        expect(SystemInfo.isAppleSubscription(managementURL: SystemInfo.appleSubscriptionsURL)) == true
+    }
+
+    func testIsAppleSubscriptionURLWithOtherStandardURL() {
+        expect(SystemInfo.isAppleSubscription(
+            managementURL: URL(string: "https://apple.com/manage_subscription")!
```

Comment:
This is what we have in `CustomerInfo` test data, and I believe what the backend returns.

Reply:
Oh gotcha I'll remove this test then.